### ISSUE TITLE
E3S10: CTH interactions against the boss are not easily discoverable in-game

### ISSUE THREAD

**Issue by shikadiqueen** [project-ethea/After_the_Storm#76]
The boss unit in this scenario has the CTH for attacks dealt against it set to 100% whenever it doesn't have any satellite units around during phases 2 and later. Periodically throughout phases 4 and 5 new mobile satellites are spawned that the player is more or less forced to kill.

However, there is no visual indication as to how the CTH interaction works during the fight. Perhaps a shield icon of some sort could be added to the boss unit's overlays whenever there are satellite units around, and give the same overlay to the satellites as a hint to the player.

**Comment by cooljeanius** [comment]
Yeah, that boss was definitely pretty cool, but I barely would've understood how anything was working if I hadn't been reading the code...
